exterminate.lua

```diff
@@ -156,13 +144,15 @@ local options, args = {
     method = killMethod.INSTANT,
     only_visible = false,
     include_friendly = false,
+    maximum = -1,
 }, {...}
 
 local positionals = argparse.processArgsGetopt(args, {
     {'h', 'help', handler = function() options.help = true end},
     {'m', 'method', handler = function(arg) options.method = killMethod[arg:upper()] end, hasArg = true},
     {'o', 'only-visible', handler = function() options.only_visible = true end},
     {'f', 'include-friendly', handler = function() options.include_friendly = true end},
+    {'x', 'maximum', handler = function(arg) options.maximum = tonumber(arg) end, hasArg = true},
```

Comment:
use `argparse.positiveInt` to sanity check format (or `argparse.nonnegativeInt` if you want to allow `0`). if the player passes `-x many`, then `options.maximum` will be set to `nil` and we'll get an exception in the `options.maximum > 0` check below.